README.md

```diff
@@ -10,7 +10,279 @@ This capsule is the the companion code to the Bixby [Space Resorts Sample Capsul
 
 ---
 
-For use cases and annotation patterns, please see [annotations](./ANNOTATIONS.md)
+## Use cases
+
+### Finding Space Resorts
+
+#### Outer "find" Queries
+
+![Outer "find" Query](./assets/images/OuterFindQuery.png)
+
+You can see all trained utterances and plans by entering this query in the
+training tab search bar: `goal:SpaceResort#all -has:continue`. Examples:
+
+- Find space resorts
+
+- Show me space hotels with crater canyoneering
+
+- Search for hotels around The Red Planet
+
+- Look for space hotels with quantum bungee jumping around Saturn
+
+We train these to have the goal `SpaceResort#all` and we annotate any resort
+names, planets and search criteria as Values. Here `SpaceResort#all` is a
+property projection to the `all` property of the `SpaceResort`. This property is
+a boolean of type `ViewAll` which is always true and is used as a proxy to
+signify that we want to display the full space resort, instead of focussing on a
+single property like `gravity` or `planet` (see property projections below). We
+use a match-pattern to then tie our views and dialogs for this property
```

Comment:
We
use a match-pattern to then tie our views and dialogs for this property
projection to describe the whole hotel.

>

A match-pattern ties the views and dialogs for this property projection to describe the whole hotel